index.js

```diff
@@ -105,7 +152,17 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let movieGenre = [];
+  for (let i = 0; i < movies.length; i++){
+    let lowerPar = genre.toLowerCase();
+    let lowerGen = movies[i].genre.toLowerCase();
+    if(lowerGen.includes(lowerPar)){
+      movieGenre.push(movies[i]);
+    }
```

Comment:
I think we could do better variable names here `lowGenre` and `lowerCurrentGenre` maybe.  Gen and Par are hard to distinguish one from the other.